src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incentives/IncentiveMigrationProperties.kt

```diff
@@ -0,0 +1,61 @@
+package uk.gov.justice.digital.hmpps.prisonerfromnomismigration.incentives
+
+import com.amazonaws.services.sqs.model.GetQueueAttributesRequest
+import com.amazonaws.services.sqs.model.GetQueueAttributesResult
+import com.amazonaws.services.sqs.model.QueueAttributeName.All
+import com.amazonaws.services.sqs.model.QueueAttributeName.ApproximateNumberOfMessages
+import com.amazonaws.services.sqs.model.QueueAttributeName.ApproximateNumberOfMessagesNotVisible
+import org.springframework.boot.actuate.info.Info.Builder
+import org.springframework.boot.actuate.info.InfoContributor
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.service.INCENTIVES_QUEUE_ID
+import uk.gov.justice.hmpps.sqs.HmppsQueue
+import uk.gov.justice.hmpps.sqs.HmppsQueueService
+
+@Component
+class IncentiveMigrationProperties(
```

Comment:
looks very similar to the visits one below, could be pulled out into superclass?

Reply:
Probably, not sure yet - depends when I do the TODO bits